ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -28,7 +28,9 @@
         {
             "cell_type": "code",
             "source": [
-                "#!eval-notebook \"Data/InitSystemorphToMemoryForTesting\""
+                "#!import \"../../ifrs17/Constants/Consts\"",
```

Comment:
1. You can't import ifr17ce like this. 
2. After importing Costs, do you still need to set ProjectionCount = 1?

Reply:
Oh yeah, of course not! we don't need it, it is set to 1 at the Consts.nb

Reply:
@dcolleoni And indeed we do need it: import doesn't assign value 1 to the ProjectionCount.

Reply:
@dcolleoni How shall we import the Consts.nb, which environment should I mention in the import line, v1.1.1 ?

Reply:
@dcolleoni It happened to be that #!import assigns a value provided inside the imported notebook only once - when run the first time inside one session. It seems to be a bug.

Reply:
@dcolleoni When we call "#!eval StringVariable", variables inside the notebook with the path StringVariable don't get assigned values. Thus we need to have additional line to set ProjectionCount = 1:
![image](https://user-images.githubusercontent.com/120557622/223695047-9c55512b-cda3-4d55-84e5-145525f814db.png)